vignettes/body_plan.Rmd

```diff
@@ -117,3 +117,8 @@ body_plan(
                                                                                  TRUE ~ frmt("x.xxx", missing = "")))
 )
 ```
+
+## Help image/s
+
+![tfrmt-frmts.jpg](images/tfrmt-frmts.jpg){width=100%}
```

Comment:
When I render the Rmd this shows up as a caption for me - are you seeing the same? If so, I say we just remove it as such:

```![](images/tfrmt-frmts.jpg){width=100%}```

Reply:
resolved in latest commit